data/assets/scene/digitaluniverse/planetarynebulae.asset

```diff
@@ -20,35 +20,28 @@ local Object = {
     Enabled = false,
     Labels = {
       File = speck .. "pn.label",
-      Color = { 0.25, 0.25, 0.65 },
+      Color = { 0.35, 0.35, 0.60 },
       Size = 16.24,
       MinMaxSize = { 4, 25 },
       Unit = "pc"
     },
     Coloring = {
       FixedColor = { 0.4, 0.4, 0.9 }
     },
-    Opacity = 0.65,
+    Opacity = 0.99,
     File = speck .. "pn.speck",
     PolygonSides = 3,
     Unit = "pc",
     SizeSettings = {
-      ScaleExponent = 18.46,
+      ScaleExponent = 17.7,
```

Comment:
If these should be visible from a perspective outside the Milky Way, this scale exponent should be reset to the old value.  

Could reduce the max size if they appear too large up close